### DISCUSSION TITLE
forEach with delay between

### DISCUSSION THREAD

**Discussion by alexei-moneyr** [postmanlabs/postman-flows#153] [Q&A] [answered]
Is there a way to make a delay between forEach iterations?

**Comment by samay-v** [comment, marked as answer]
Hey @alexei-moneyr 
You can add the `Delay` block after the `For Each` block to achieve this. 
It would look something like this 
![image](https://user-images.githubusercontent.com/28357106/145382936-901eb5e3-204e-4bb1-84ef-44a1c3866774.png)

**Reply by alexei-moneyr** [reply]
My bad, I thought that it will work when whole forEach block ends. 
Big thanks
